modules/ROOT/pages/tools/neo4j-admin/migrate-database.adoc

```diff
@@ -83,7 +83,7 @@ If not specified, the tool migrates the target database to the latest known comb
 
```

Comment:
About the `--force-btree-indexes-to-range` option, that is only useful for a migration from 4.4, and we are not officially supporting going from 4.4 -> 2025.01 (should go through 5.26)
I have added a card for kernel to deprecate the option on dev, but I think we should update the description here too. It should at the very least not be referencing v5 migration guide I think since it is in the 2025 docs
Maybe say something about it only being used when migrating from 4.4?

Reply:
Yes, I saw this option is not relevant for 2025, but decided not to change it as the descriptions are tested in the docs.